src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2310,6 +2310,26 @@ void processContextElement
                     getEntitiesCollectionName(tenant).c_str(),
                     query.toString().c_str()));
 
+    std::map<std::string, std::string>::iterator it = uriParams.find("op");
```

Comment:
Have you tried `make unit_test` in this branch? I'm getting a segfault in mongoNotifyContextRequest.Ent1Attr1 (but it could be a false positive, which only happens in my system...).

In addition to `make unit_test`, `make functional_test` is also needed as "regression" in order to check that at PR is not breaking any existing functionality (this is standard Orion way-of-working, don't worry, it's easy to learn :)